ts_src/psbt.ts

```diff
@@ -1759,12 +1762,8 @@ export interface UnblindWitnessUtxoResult {
 
 function getBlindingDataForInput(
   prevout: WitnessUtxo,
-  blindPrivKey?: Buffer,
-): UnblindWitnessUtxoResult | undefined {
-  if (!blindPrivKey) {
-    return undefined;
-  }
-
+  blindPrivKey: Buffer,
+): UnblindWitnessUtxoResult {
```

Comment:
I don't really like this name because it seems that blinding/unblinding is supported only for segwit/native-segwit outputs this way. I would call this `UnblindPrevoutResult`.. eventually, since this is an internal method, it could simply return `any` given that the mentioned type is just a wrapper of the confidential module's method (we should plan to fix the confidential module so that it returns more "user-friendly"er results... like we did in go-elements basically.